### ISSUE TITLE
It causes Extension Host Failed error

### ISSUE THREAD

**Issue by psychoticSJ** [microsoft/azuredatastudio#24981]
Type: <b>Bug</b>

I used Extension Bisect to run through my extensions to see when the error would come. it only came when this extension was enabled. i have no idea what's causing it. please help. 

Extension version: 0.5.3
Azure Data Studio version: azuredatastudio 1.47.0 (c7c2b2f21505562d21972d4c135119d00806db4f, 2023-11-07T17:17:45.669Z)
OS version: Windows_NT x64 10.0.22631
Restricted Mode: No
Preview Features: Enabled
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|12th Gen Intel(R) Core(TM) i9-12900HK (20 x 2918)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|63.69GB (46.37GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|
</details>

**Comment by kisantia** [comment]
@psychoticSJ can you please share the link to the Bisect extension? I don't see it in the ADS extensions marketplace

**Comment by Charles-Gagnon** [comment]
@kisantia I believe they're talking about the "Extension Bisect" feature : 

![image](https://github.com/microsoft/azuredatastudio/assets/28519865/20278dc7-3903-4f71-b698-01ed81b356e9)

Given the extension version number it seems likely that this is an issue with the query history extension - is that correct @psychoticSJ ? Could you explain a bit more what the error is and/or what behavior you're seeing that seems incorrect?

**Comment by psychoticSJ** [comment]
Yes that's correct. It's the query history extension that was causing an issue.
It would say, "extension host stopped working" or something like that. Let me see if i can re-enable the extension so that the error comes up again. Then I'll take a screenshot.

Is that ok?

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by psychoticSJ** [comment]
It also feels as if it made the Github Copilot extension fail to install. Because only after i disabled it was i able to install it.

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
